packages/e2e-tests/fixtures/blocks/core__table-of-contents.serialized.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:table-of-contents -->
+<div class="wp-block-table-of-contents"><ul><li><a href="#0-First-Heading" data-level="2">First Heading</a><ul><li><a href="#1-Sub-Heading" data-level="3">Sub Heading</a></li><li><a href="#2-Another-Sub-Heading" data-level="3">Another Sub Heading</a></li></ul></li></ul></div>
```

Comment:
Why it's not a `<nav>` tag instead of `<div>` container ?

Reply:
Ah yes, this is semantically correct. I will make this change